ocpp-transport/src/main/kotlin/com/izivia/ocpp/transport/OcppVersion.kt

```diff
@@ -0,0 +1,5 @@
+package com.izivia.ocpp.transport
+
+enum class OcppVersion(val subprotocol: String) {
```

Comment:
We have 2 `OcppVersion` enum, one in `transport` and one in `ocpp-wamp`. 
The enum in `transport` is for Apis and the interface `ServerTransport`, unlike the one defined in the module `ocpp-wamp` which is proper to this particular transport.
I can put it at the root, and we would use the same enum for all the modules, `ocpp-wamp` included

Reply:
We have 2 enum OcppVersion because ocpp-wamp doesn't depends of any other module so the enum need to be in ocpp-wamp, and the other modules doesn't necessary depends of ocpp-wamp so we have a second OcppVersion for these modules.